tests/suites/host_test.function

```diff
@@ -601,7 +601,7 @@ int execute_tests( int argc , const char ** argv )
     }
 
     /* Initialize the struct that holds information about the last test */
-    memset( &mbedtls_test_info, 0, sizeof( mbedtls_test_info ) );
+    mbedtls_test_info_reset();
```

Comment:
Not possible to just remove this?

Reply:
Why? The information from the previous test does need to be erased before running the next test, and this seems like a good place for it.

Reply:
I am not sure about that and I am probably missing/not understanding something here. I don't see where/how a previous test is executed. The test cases in this function are executed below this point. Previous call to execute_tests()? But execute_tests() seems to be called only once from main().

Reply:
I don't think so because there is a usage of `mbedtls_test_info.result` at line 641 before the next `mbedtls_test_info_reset()`.

Reply:
Ah, I wasn't paying attention to the context. I think here  the memset was redundant since it was just duplicating static initialization. (We assume that all-bits-zero and logical zero are semantically equivalent throughout the library.) But now that there's a function, it should be called here, since it change to setting values other than all-zero.

Reply:
Given your two last comments yes I agree we should keep it. Thanks.